docs/configuration/upstream/propose_downstream.md

```diff
@@ -3,10 +3,16 @@ title: Propose downstream
 sidebar_position: 5
 ---
 
-# `job: propose_downstream`
+# `job: propose_downstream` 
 
-Land a new upstream release in Fedora. This job only makes sure the changes
-happen in Fedora dist-git - no builds. A pull request is created as a result.
+Land a new upstream release in Fedora (or CentOS Stream). This job only makes sure the changes
+happen in dist-git - no builds. A pull request is created as a result.
+
+:::tip CentOS Stream support
+
+We newly support `propose-downstream` also for CentOS Stream, see the configuration specifics in the 
+[subsection below](#syncing-the-release-to-centos-stream).
+:::
```

Comment:
```suggestion
:::tip CentOS Stream support

We newly support `propose-downstream` also for CentOS Stream, see the configuration specifics in the 
[subsection below](#syncing-the-release-to-centos-stream).

:::
```

Empty lines are because of `prettier` potentially turning:
```
:::tip
something
:::
```
into
```
:::tip something :::
```
which breaks the parser…